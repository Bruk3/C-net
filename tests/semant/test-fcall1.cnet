int main() {
    int a = 5; 
    int b = y(x(a)) + 'a';

    return 0;
}

int x(int b) {return b;}
int y(int c) {return c;}
