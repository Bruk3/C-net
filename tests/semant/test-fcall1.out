int main()
{
{
int a = (int : 0);
(int : a=(int : 5));
}
{
int b = (int : 0);
(int : b=(int : y((int : x((int : a))))));
}
return (int : 0);
}

int x(int b)
{
return (int : b);
}

int y(int c)
{
return (int : c);
}

